test/benchmark/LockupLinear.Gas.t.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.22;
+
+import { LockupLinear } from "src/types/DataTypes.sol";
+
+import { Benchmark_Test } from "./Benchmark.t.sol";
+
+/// @notice Benchmark Test for the LockupLinear contract.
+/// @dev This contract creates a markdown file with the gas usage of each function in the benchmarks directory.
+contract LockupLinear_Gas_Test is Benchmark_Test {
+    /*//////////////////////////////////////////////////////////////////////////
+                                  SET-UP FUNCTION
+    //////////////////////////////////////////////////////////////////////////*/
+    function setUp() public override {
+        super.setUp();
+
+        lockup = lockupLinear;
+    }
+
+    /*//////////////////////////////////////////////////////////////////////////
+                                   TEST FUNCTION
+    //////////////////////////////////////////////////////////////////////////*/
+
+    function testGas_Implementations() external {
+        // Set the file path
+        benchmarksFile = string.concat(benchmarksDir, "SablierV2LockupLinear.md");
+
+        // Create the file if it doesn't exist, otherwise overwrite it
+        vm.writeFile({
+            path: benchmarksFile,
+            data: string.concat(
+                "# Benchmarks for implementations in the LockupLinear contract\n\n",
+                "| Implementation | Gas Usage |\n",
+                "| --- | --- |\n"
+            )
+        });
+
+        // Set the caller to recipient for `burn` and change timestamp to end time
```

Comment:
We usually refer to the test recipient user as the `Recipient`. Ditto for the sender user, who is the `Sender`.